Libs/Visualization/VTK/Widgets/ctkVTKChartView.cpp

```diff
@@ -74,12 +76,51 @@ ctkVTKChartViewPrivate::ctkVTKChartViewPrivate(ctkVTKChartView& object)
 #ifdef CTK_USE_QVTKOPENGLWIDGET
   this->RenderWindow = vtkSmartPointer<vtkGenericOpenGLRenderWindow>::New();
 #endif
-  this->Chart = vtkSmartPointer<vtkChartXY>::New();
-  this->ContextView->GetScene()->AddItem(this->Chart);
-  this->UserBounds[0] = this->UserBounds[2] = this->UserBounds[4] = this->UserBounds[6] = 0.;
-  this->UserBounds[1] = this->UserBounds[3] = this->UserBounds[5] = this->UserBounds[7] = -1.;
-  this->OldBounds[0] = this->OldBounds[2] = this->OldBounds[4] = this->OldBounds[6] = 0.;
-  this->OldBounds[1] = this->OldBounds[3] = this->OldBounds[5] = this->OldBounds[7] = -1.;
+  object.setChartType(ctkVTKChartView::ChartTypes::XYChart);
```

Comment:
Is this enum Python-wrapped?

Reply:
I believe that yes, thanks to Q_INVOKABLE.